### PR TITLE
Adjust guess logic to account for repeating letters

### PR DESCRIPTION
**Pull request by lunchkin**

### CODE REVIEW

File: src/components/game.tsx [hunk 1]
```diff
@@ -4,10 +4,18 @@ import { TileStatus } from "../components/game-tile";
 import { TileState } from '../types/tile-state';
 import GameRow from './game-row';
 
-
-const CORRECT_WORD = 'CRANE';
+const CORRECT_WORD = 'THREE';
 const CORRECT_LETTERS = CORRECT_WORD.split('');
 
+const CORRECT_WORD_HASH = CORRECT_LETTERS.reduce(
+    (acc, letter) => {
```
Comment (acantrell) [inline src/components/game.tsx:11]: **suggestion (non-blocking):** ‏you could add types here

```suggestion
    (acc: { [letter: string]: number }, letter: string) => {
```

File: src/components/game.tsx [hunk 2]
```diff
@@ -4,10 +4,18 @@ import { TileStatus } from "../components/game-tile";
...
+        acc[letter] ? acc[letter]++ : acc[letter] = 1;
+
+        return acc;
+    }, 
+    {} as any
```
Comment (acantrell) [inline src/components/game.tsx:16]: **suggestion (non-blocking):** ‏type this

```suggestion
    {} as { [letter: string]: number }
```

**Review by acantrell — APPROVED** [review]
✅ Approved w/ typing suggestions

**Review by nlaveck — APPROVED** [review]
LGTM 👍

**Pull request merged**